tensorflow/lite/python/interpreter_test.py

```diff
@@ -113,6 +113,21 @@ def testInvokeBeforeReady(self):
                                  'Invoke called on model that is not ready'):
       interpreter.invoke()
 
+  def testInvalidModelFileContent(self):
+    with self.assertRaisesRegexp(
+        ValueError, '`model_path` or `model_content` must be specified.'):
+      interpreter_wrapper.Interpreter(model_path=None, model_content=None)
+
+  def testInvalidindex(self):
+    interpreter = interpreter_wrapper.Interpreter(
+        model_path=resource_loader.get_path_to_datafile(
+            'testdata/permute_float.tflite'))
+    interpreter.allocate_tensors()
+    #Invalid tensor index passed
```

Comment:
# Invalid tensor index passed.

Reply:
@jdduke , thanks for the review, i have updated the code as per suggestion. Kindly check.

Regards
Amit